dogu.json

```diff
@@ -127,6 +127,96 @@
       "Validation": {
         "Type": "FLOAT_PERCENTAGE_HUNDRED"
       }
+    },
+    {
+      "Name": "cleanup_policy/notes",
+      "Description": "Notes to the default cleanup-policy.",
+      "Optional": true,
+      "Default": "Default policy that is generated by the ces via a script on nexus startup"
+    },
+    {
+      "Name": "cleanup_policy/mode",
+      "Description": "Mode to the default cleanup-policy.",
+      "Optional": true,
+      "Default": "deletion"
```

Comment:
I wonder why here is no validateable list of values. After some research I come to the conclusion, that there is only on possible value: `delete`.

This block feels like a candidate for a hardcoded value. What do you think?

Reply:
Yes the API is unfortunately [very poorly documented](https://www.mvndoc.com/c/org.sonatype.nexus/nexus-cleanup/org/sonatype/nexus/cleanup/storage/CleanupPolicy.html#setMode-java.lang.String-) in this regard. I have created several policies in the frontend and checked what is in their payload. Unfortunately the "mode" is not part of the fields. But this supports the thesis that only `deletion` can be specified statically.

I also found a [very old documentation (v2)](https://nexus3-cli.readthedocs.io/_/downloads/en/2.3.1/pdf/) of the nexus-cli in which also only "delete" can be specified as mode.